samples/unit-tests/chart/panning/demo.js

```diff
@@ -726,3 +726,43 @@ QUnit.test(
         assert.strictEqual(actualMin, -75, 'Min must be -75; not panning');
     }
 );
+
+QUnit.test('Column chart panning.', function (assert) {
```

Comment:
Could you move the test to `Ordinal axis panning`? It seems to be almost exactly the same with some minor changes.